tests/tests.mk

```diff
@@ -2,9 +2,13 @@ export TIMEOUT ?= 30m
 
 # gocache disabled by -count=1
 # tests in different packages forced to be sequential by -p=1
-doc.test: init
-	@${GO} test ${REPO_ROOT}/tests/setup/... -v -timeout=${TIMEOUT} -count=1 -p=1 \
-		-istio.test.env=kube -istio.test.hub=$(HUB) -istio.test.tag=$(TAG)
+doc.test: init | $(JUNIT_REPORT)
+	@${GO} test ${REPO_ROOT}/tests/setup/... \
+		-v -timeout=${TIMEOUT} -count=1 -p=1 \
+		-istio.test.env=kube \
```

Comment:
We can probably remove the env line since kube is now the only option.

Reply:
Okay I removed this and hopefully it will pass the k8s-tests